contract_sale_generation/models/contract_line.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ContractLine(models.Model):
+    _inherit = "contract.line"
+
+    def _prepare_sale_line_vals(self, dates, order_id=False):
```

Comment:
What is the use of the `order_id` parameter and logic? It seems to be called nowhere with `order_id=True`.